sql/updates/1.7.5--2.0.0-rc1.sql

```diff
@@ -264,6 +258,16 @@ DROP VIEW IF EXISTS timescaledb_information.continuous_aggregates;
 DROP VIEW IF EXISTS timescaledb_information.continuous_aggregate_stats;
 ALTER TABLE IF EXISTS _timescaledb_catalog.continuous_agg DROP COLUMN IF EXISTS job_id;
 
+-- Save ACL for tables that are being rebuilt
+CREATE TABLE saved_acl(tmpoid oid, tmpns oid, tmpacl aclitem[]);
```

Comment:
Could this be a temporary table?

Reply:
Using a temporary table here does not offer any advantage. They are dropped at the end of the session, not at the end of the update, so they need to be explicitly removed anyway.